packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -2261,8 +2262,17 @@ export default class Meeting extends StatelessWebexPlugin {
         });
       }
     });
-    this.locusInfo.on(LOCUSINFO.EVENTS.SELF_ADMITTED_GUEST, (payload) => {
+    this.locusInfo.on(LOCUSINFO.EVENTS.SELF_ADMITTED_GUEST, async (payload) => {
       this.stopKeepAlive();
+      // @ts-ignore
+      if (!this.transcription && (this.config.receiveTranscription || this.receiveTranscription)) {
```

Comment:
Question: Why do we have two different checks `this.config.receiveTranscription` and `this.receiveTranscription`?

Reply:
`this.config.receiveTranscription` checks if we have a `receiveTranscription` property when we initialise SDK with config object.

`this.receiveTranscription` is when user selects `startsReceivingTranscription()` in samples app before joining a meeting means if we are passing `receiveTranscription: true` in join meeting options.